utility/src/main/java/org/oppia/android/util/logging/firebase/FirebaseEventLogger.kt

```diff
@@ -19,7 +21,7 @@ class FirebaseEventLogger private constructor(
   private val firebaseAnalytics: FirebaseAnalytics,
   private val networkConnectionUtil: NetworkConnectionUtil,
   private val eventBundleCreator: EventBundleCreator
-) : EventLogger {
+) : EventLogger, PerformanceMetricsEventLogger {
```

Comment:
How painful would it be to rename the existing event logger? I'm just a bit concerned that it's too general sounding now that we're specializing performance metrics.

Reply:
We'll need to do the following: 
- changes in all the production files where eventLogger is used,
- changes in test files. 
- changes in dagger modules. 

It might not be too painful -- we can file this issue as a good-second-issue.

Reply:
Filed an issue for this -- https://github.com/oppia/oppia-android/issues/4405

Reply:
That seems reasonable--thanks for filing the issue!